mintpy/timeseries2velocity.py

```diff
@@ -324,27 +337,48 @@ def read_inps2model(inps):
     if 'polynomial' not in model.keys():
         raise ValueError('linear/polynomial model is NOT included! Are you sure?!')
 
+    if len(model['exp']) % 2 != 0:
+        raise ValueError('num of exponential model inputs is odd! Specify each exp func with first a onset date AND followed by a char time.\n'+
+                        '\tE.g., --exp  20181026 60 \n' +
+                        '\t   or --exp  201601231 80.5 20190125 200  (append as many exps as you like)')
+    else:
+        tmp_arr = np.reshape(model['exp'], (int(len(model['exp'])/2), 2))
+        model['exp'] = [list(tmp_arr[:,0]), list(tmp_arr[:,1].astype(float))]
+
+    if len(model['log']) % 2 != 0:
+        raise ValueError('num of logarithmic model inputs is odd! Specify each log func with first a onset date AND followed by a char time.\n'+
+                        '\tE.g., --log  20181026 60 \n' +
+                        '\t   or --log  20161231 80.5 20190125 200  (append as many logs as you like)')
+    else:
+        tmp_arr = np.reshape(model['log'], (int(len(model['log'])/2), 2))
+        model['log'] = [list(tmp_arr[:,0]), list(tmp_arr[:,1].astype(float))]
+
```

Comment:
This part of the code should belong to `cmd_line_parse()`.

Reply:
They are now moved to `cmd_line_parse()`. Please look at the new commit.